src/app/services/system-general.service.ts

```diff
@@ -48,12 +52,18 @@ export class SystemGeneralService {
     combineLatest([
       this.isStable(),
       this.store$.pipe(waitForSystemInfo),
-    ]).subscribe(([isStable, sysInfo]) => {
+      this.store$.select(selectSystemHostId).pipe(filter(Boolean)),
+      this.sessionId$,
+    ]).subscribe(([isStable, sysInfo, hostId, sessionId]) => {
       if (!isStable) {
         Sentry.init({
           dsn: environment.sentryPublicDsn,
           release: sysInfo.version,
         });
+        Sentry.configureScope((scope) => {
```

Comment:
I think it's time to create a new service.

Reply:
Good idea 👍🏻